### PR TITLE
 Add keybindings for resizing frames

### PR DESCRIPTION
**Pull request by djbarnwal**
This implements `Ctrl+left` and `Ctrl+right` keybindings which help you resize the two frames in steps of 0%, 33%, 50%, 66% and 100% .

(Mirror of #754)

### CODE REVIEW

**Comment by djbarnwal** [comment]
@hamilton This PR is ready for another review.

File: src/eval-frame/actions/eval-frame-tasks.js [hunk 1]
```diff
@@ -39,6 +39,20 @@ tasks.closePanes = new UserTask({
   },
 })
 
+tasks.increaseEditorWidth = new UserTask({
+  title: 'Set Editor Size',
+  keybindings: ['ctrl+right'],
```
Comment (hamilton) [inline src/eval-frame/actions/eval-frame-tasks.js]: this should be `ctrl+shift+right` as well - if I am focused on the iframe I lose the keybinding

File: src/port-to-eval-frame.js [hunk 2]
```diff
@@ -25,6 +25,8 @@ const approvedKeys = [
   'ctrl+d',
   'ctrl+h',
   'ctrl+i',
+  'ctrl+left',
```
Comment (hamilton) [inline src/port-to-eval-frame.js]: same as above - should be `ctrl+shift+right` etc.

**Comment by djbarnwal** [comment]
@hamilton I think the Help modal error is due to some other commit. I posted an issue regarding the same ( #840 ) few hours ago.

File: src/components/editor-pane-container.jsx [hunk 3]
```diff
@@ -53,15 +54,15 @@ class EditorPaneContainer extends React.Component {
...
+          minWidth={this.props.paneMinWidth}
           onResizeStop={this.onResizeStopHandler}
           size={{ width: this.props.cellResizerWidth }}
           style={{
             display: this.props.cellResizerDisplayStyle,
             flexDirection: 'column',
           }}
         >
-          <div id="cells">
+          <div style={this.props.cellsStyle} id="cells">
```
Comment (hamilton) [inline src/components/editor-pane-container.jsx]: @bcolloran do you have any thoughts on this approach?

Comment (bcolloran) [reply src/components/editor-pane-container.jsx]: @hamilton seems reasonable enough to me, do you have any particular concern?

**Comment by bcolloran** [comment]
look good to me. i'm happy to merge if @hamilton is ok with the ` style={this.props.cellsStyle} ` part

**Pull request merged**